visualization-project-laptop-sales/visualization_project_laptop_sales_sol.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""visualization-project-laptop-sales_sol
+
+Automatically generated by Colab.
+
+Original file is located at
+    https://colab.research.google.com/drive/10M53HHyac1sf2zjJ_UkB59movx5UKI0y
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import datetime
+
+"""# **#1 What is target market? Male or female?**"""
+
+sales = pd.read_csv(r'/content/Sales_data.txt')
+sales
+
+# Comparing 'Profit' between male and female
+total_profit = sales.groupby('Contact Sex')['Profit'].sum()
+total_profit
+
+"""1. The target marget seems to be males since their total profit of 2936.53 is higher than of females, which is 2522.48.
+
+# **#2 If business is cash-constrained, which gender should be targeted?**
+"""
+
+# Let's look at the 'Our Cost' column
+
+business_cost = sales.groupby('Contact Sex')['Our Cost'].sum()
```

Comment:
Try also including "Shipping Cost" since we don't know if that will fall on the business, consumer, or both.

Reply:
Also this should be looking at Profit/Cost